Makefile

```diff
@@ -9,7 +9,9 @@ CFLAGS += -Wno-unused-label
 CFLAGS += -include src/common.h
 
 # Set the default stack pointer
-CFLAGS += -D DEFAULT_STACK_ADDR=0xFFFFF000
+CFLAGS += -D DEFAULT_STACK_ADDR=0xFFFE000
```

Comment:
> Why you remove the highest 4 bit?

Left some space(4096) for list of arguments. According to the [ARG_MAX](https://man7.org/linux/man-pages/man3/sysconf.3.html), this size list of arguments must not be less than _POSIX_ARG_MAX (4096). Since we are emulating target program, so we can decide our own size instead and I think 4096 is decent for now.

Reply:
Your reply seems like you change `F000` to `E000`, but my question is why you change `FFFFF000` to `FFFF000`

Reply:
> Your reply seems like you change `F000` to `E000`, but my question is why you change `FFFFF000` to `FFFF000`. Maybe I misunderstand your reply, could you explain it more detail.

My bad, I will change it back to 'FFFFF000'